library/src/blas1/rocblas_amax_amin.h

```diff
@@ -39,9 +30,11 @@ struct rocblas_default_value<index_value_t<T>>
     }
 };
 
-// Fetch absolute value
+//!
+//! @brief Struct-operator to fetch absolute value
+//!
 template <typename To>
-struct rocblas_fetch_amax_amin
+struct rocblas_fetch_index_value
```

Comment:
`amax_amin` seems more intuitive than `index_value`, if the `struct` you are passing is supposed to designate "how to fetch the values for <insert_function_here>", i.e., "How to fetch the values for `amax`, `amin`," as opposed to `asum` or `nrm2`, makes more sense than"how to fetch the values for `index_value`" (whatever that means). `index_value` is an internal implementation detail which does not need to be exposed when expressing `amax`, `amin`.